link preview style changes

adianasct opened the pull request:
link preview style changes

adianasct committed:
style adjustments

adianasct committed:
Enhance MessageBody and OGMetadata components with improved link handling and styling

- Added logic to determine if a link attachment URL is long, affecting layout and rendering.
- Updated MessageBody to conditionally render attachment widths based on link attachment presence and length.
- Refactored OGMetadata to control image display based on calculated height, ensuring better visual consistency.
- Adjusted styling for various elements to improve overall UI and maintainability.

adianasct committed:
change 470px width to 400 then url is long

adianasct committed:
Enhance MessageBody component to calculate and apply attachment height

- Introduced a new useMemo hook to calculate the height of media attachments based on their dimensions and type.
- Updated MessageBodyContainer styling to conditionally set max-height based on the calculated attachment height, improving layout consistency.
- Ensured that the attachmentHeight prop is passed correctly to maintain responsive design.

adianasct committed:
remove the long urls wrapping functionality

adianasct committed:
Merge branch 'feature/poll-messages' of https://github.com/sceyt/sceyt-chat-react-uikit into fix/link-preview-fixes

adianasct committed:
Merge branch 'feature/poll-messages' of https://github.com/sceyt/sceyt-chat-react-uikit into fix/link-preview-fixes

adianasct committed:
styles changed

karensct commented on src/components/Message/MessageBody/index.tsx:
```diff
@@ -432,6 +457,51 @@ const MessageBody = ({
   )
   const selectionIsActive = useMemo(() => selectedMessagesMap && selectedMessagesMap.size > 0, [selectedMessagesMap])
 
+  const hasLongLinkAttachmentUrl = useMemo(() => {
```
@adianasct this will cut all links doesn't matter is it link preview or no

adianasct committed:
Update OGMetadata component to enhance metadata callback and styling

- Modified metadataGetSuccessCallback to include metadata parameter for improved data handling.
- Added textPrimary color to Title component for better visual consistency.
- Cleaned up unused console log and adjusted useEffect dependencies for optimized performance.

karensct committed:
Merge branch 'feature/poll-messages' of https://github.com/sceyt/sceyt-chat-react-uikit into fix/link-preview-fixes

karensct committed:
Merge branch 'feature/poll-messages' of https://github.com/sceyt/sceyt-chat-react-uikit into fix/link-preview-fixes

karensct committed:
Refactor message delivery status handling and clean up unused code

- Moved the delivery status check logic into the Message component to ensure accurate updates for delivered and read messages.
- Removed redundant code related to attachment height calculations in MessageBody component for improved readability and performance.
- Updated message reducer to streamline delivery status updates based on existing statuses.
